tap_quickbooks/schemas/items.json

```diff
@@ -77,15 +77,17 @@
       ]
     },
     "QtyOnHand": {
```

Comment:
If it is quantity then it won't be in decimal.

Reply:
Reverted back QtyOnHand to an integer.